tests/e2e/support/objects/app-files/link/actions.ts

```diff
@@ -67,8 +67,11 @@ const publicLinkEditRoleButton =
   `//h4[contains(@class, "oc-files-file-link-name") and text()="%s"]//ancestor::li//div[contains(@class, "link-details")]/` +
   `div/button[contains(@class, "edit-public-link-role-dropdown-toggle")]`
 const addPublicLinkButton = '#files-file-link-add'
-const getMostRecentLink = '//div[@id="oc-files-file-link"]//ul/li[1]'
-const publicLink = `//ul/li/div/h4[contains(text(),'%s')]/following-sibling::div//p`
+const publicLinkNameList =
+  '//div[@id="oc-files-file-link"]//ul//h4[contains(@class,"oc-files-file-link-name")]'
+const publicLinkUrlList =
+  '//div[@id="oc-files-file-link"]//ul//p[contains(@class,"oc-files-file-link-url")]'
+const publicLink = `//ul/li/div/h4[text()='%s']/following-sibling::div//p`
```

Comment:
i think we can simplify this xpath

Reply:
probably the last one can be. good to have a scope though
```diff
-const publicLink = `//ul/li/div/h4[text()='%s']/following-sibling::div//p`
+const publicLink = `//ul//h4[text()='%s']/following-sibling::div//p`
```